server/index.js

```diff
@@ -1,72 +1,93 @@
-const
+const path = require('path'),
```

Comment:
**Manual fix:**
**LN 4 - Resolved:** Updated imports to put path first. Should have no impact.